packages/components/src/custom-select-control/index.js

```diff
@@ -187,10 +187,10 @@ export default function CustomSelectControl( props ) {
 							items.map( ( item, index ) => (
 								// eslint-disable-next-line react/jsx-key
 								<li
+									key={ item.key }
```

Comment:
I'm guessing this wasn't caught by the React 18.3 update because we're using the old JSX transform.

Reply:
yes